programs/program-statistics.md

```diff
@@ -0,0 +1,21 @@
+# Program Statistics
+
+| Year | Program                          | Internships | Total (per year) |
+| ---- | -------------------------------- | ----------- | ---------------- |
+| 2022 | LFX Mentorship (CommunityBridge) | TBD         | TBD              |
```

Comment:
We have the numbers now for 2022, correct?

Reply:
🙏